src/covid_shared/cli_tools/decorators.py

```diff
@@ -169,3 +169,11 @@ def add_snapshot_dependency_option():
     Declares a dependency on a specific snapshot version. Defaults to "best".
     """
     return _create_dependency_option('snapshot')
+
+
+def validate_options_with_q(quick: int, mark_best: bool, production_tag: str):
```

Comment:
This isn't a decorator so I don't think it should live in the decorators module. Or we should rename the module to more accurately reflect its meaning.

Should we add a validation module inside cli_tools directory?